bundle/config/mutator/paths/pipeline_paths_visitor.go

```diff
@@ -1,12 +1,14 @@
 package paths
 
 import (
+	"github.com/databricks/cli/bundle/libraries"
 	"github.com/databricks/cli/libs/dyn"
 )
 
 type pipelineRewritePattern struct {
-	pattern dyn.Pattern
-	mode    TranslateMode
+	pattern     dyn.Pattern
+	mode        TranslateMode
+	skipRewrite func(string) bool
```

Comment:
What is this for, can you add documentation? It's always noSkipRewrite, is that intentional?

Reply:
It's used here to skip rewrite :) https://github.com/databricks/cli/pull/3153/files/e761703ccd5f6007ae8365d49308c884ac74c732#diff-247ed09a8450b38c98113c320bc9aeadbc829d6a6699ee51ee748e550ffd8f4fR33